docs/triplydb-js/service/index.md

```diff
@@ -84,12 +84,14 @@ console.log(await service.isUpToDate())
 ```
 
 
-## Service.update()
+## Service.update( opts?)
 
 Synchronizes the service. Synchronization means that the data that is used in the service is made consistent with the data that is present in the graphs of the dataset.
 
 When one or more graphs are added or deleted, existing services keep exposing the old state of the data. The changes in the data are only exposed in the services after synchronization is performed.
 
+You can choose to perform a rolling update, during which a new service is replacing the old one, in order to eliminate downtime. For a rolling update you should use the object `{rollingUpdate:true}` as `opts`. However, be aware that when using a rolling update, depending on your dataset statements, you might reach your instance's limit of statements' number for services. In this case, you will not be able to do a successful rolling update and you should switch to a regular update.
```

Comment:
the reason why I didn't put it is that it's a bit advanced, and also I didn't want to give very explicit details on how to use it: I would have to say that:
- the user can log the progress using a loging function as the value of onProgress argument AND
- the user can also change the progress messages for each stage
All this makes doc very verbose and I wanted to avoid this.
I guess I can do it halfway, and just mention only the first part, the the user can print progress messages by using a logging function in the `onProgress` argument. What do you think about that?

Reply:
https://github.com/TriplyDB/Documentation/pull/124/commits/2e6baccd3b452b6096568f48c24fd4ac21969db2